spec/models/user_spec.rb

```diff
@@ -121,4 +121,15 @@
       expect { subject }.to change { member.state }.from("key_member").to("voting_member")
     end
   end
+
+  describe "#door_code" do
+    subject { create(:user) }
+
+    it { is_expected.to have_one(:door_code) }
+
+    it "defaults to nil" do
+      new_user = User.create!(name: "Kay Doke", email: "k@ok.com", username: "k")
```

Comment:
http://ok.com/ is a real site, I would prefer to use http://example.com/ even in tests where it doesn't matter 😅